yarn.lock

```diff
@@ -7242,7 +7247,7 @@ lodash.memoize@^4.1.2:
   resolved "https://registry.yarnpkg.com/lodash.memoize/-/lodash.memoize-4.1.2.tgz#bcc6c49a42a2840ed997f323eada5ecd182e0bfe"
   integrity sha1-vMbEmkKihA7Zl/Mj6tpezRguC/4=
 
-lodash.merge@^4.6.1:
+lodash.merge@^4.0.0, lodash.merge@^4.6.1:
```

Comment:
can we dedupe this?

Reply:
I don't understand the question.  v4.6.2 is used by everything.